.github/workflows/deploy.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       run: echo version=${GITHUB_REF#refs\/tags\/v} >> $GITHUB_OUTPUT
 
     - name: Build and deploy to Clojars
-      uses: yetanalytics/actions/deploy-clojars@v0.0.4
```

Comment:
You're actually supposed to use version 1.1.0 (since from the user's perspective there's no breaking change in the params or anything). I can delete the 2.0.0-SNAPSHOT tag for the action if it's too confusing.

Reply:
Alternatively you can use v1 since I mutated that tag (as I've seen other GH action maintainers do)

Reply:
done